### PR TITLE
Milestone 1

### PR DESCRIPTION
**Pull request by ZJUAlexChen**
This is for the first Pull Request with readme file finished.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,235 @@
+# Question-answering-with-extracted-text-summarization
+This project is to answer questions related to keywords using google with extracted text summarization.
+
+# Functional Design
```
Comment (ashuk203) [inline README.md:4]: For you functional design, please use `snake_case` for your python functions (you are currently using `camelCase`)

File: README.md [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+### Function 2 trainModel
+* ***Functionality***: Train the model if it is necessary for the user
+* ***Input***:
+   * referenceText: standard reference text is needed for calculation of saliency scores
+   * data: data with sentences from original text
+   * word2vecWordNum: number of words used in word embedding with word2vec
+   * wordNum: maximum number of words to keep (with relatively large frequency)
+* ***Output***: trained model
+```python
+def trainModel( referenceText, data, word2vecWordNum, wordNum ):
```
Comment (ashuk203) [inline README.md]: Could you provide add more detail / clarification of what the `data` argument? Is it the training data?

**Review by ashuk203 — CHANGES_REQUESTED** [review]
Apart from the few changes requested in the comments, everything looks good!

File: README.md (hunk 1 above)
Comment (ZJUAlexChen) [reply README.md:4]: I have changed the camelCase to snake_case. Thanks.

File: README.md (hunk 2 above)
Comment (ZJUAlexChen) [reply README.md]: I have clarified the data argument as the training data. Thanks. I wonder if I should merge the request by myself after your second review, or you merge it after reviewing my correction?

**Review by ashuk203 — APPROVED** [review]

**Comment by ashuk203** [comment]
You can go ahead and merge your pull request. Good work!

**Pull request merged**